test/Microsoft.Azure.WebJobs.Host.EndToEndTests/InvocationFilterEndToEndTests.cs

```diff
@@ -0,0 +1,635 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Host.EndToEndTests
+{
+    public class InvocationFilterEndToEndTests : IClassFixture<TestFixture>
+    {
+        private JobHostConfiguration _config;
+        private JobHost _host;
+
+        private const string TestArtifactPrefix = "e2etestmultiaccount";
+        private const string Input = TestArtifactPrefix + "-input-%rnd%";
+        private const string Output = TestArtifactPrefix + "-output-%rnd%";
+        private const string InputTableName = TestArtifactPrefix + "tableinput%rnd%";
+        private const string OutputTableName = TestArtifactPrefix + "tableinput%rnd%";
+        private const string TestData = "﻿TestData";
+        private const string Secondary = "SecondaryStorage";
+        private readonly TestFixture _fixture;
+        private readonly TestLoggerProvider _loggerProvider = new TestLoggerProvider();
+
+        public InvocationFilterEndToEndTests(TestFixture fixture)
+        {
+            _config = new JobHostConfiguration()
+            {
+                TypeLocator = new FakeTypeLocator(typeof(AsyncChainEndToEndTests))
+            };
+
+            ILoggerFactory loggerFactory = new LoggerFactory();
```

Comment:
Move all this into your TestFixture class below, so the fixture Host property returns this host.